solutions/usaco-1159.mdx

```diff
@@ -86,11 +90,11 @@ public class ConnectingTwoBarns {
 				int dist = Math.abs(barn1.get(barn1Index) - i);
 
 				/*
```

Comment:
Looks like the multi line comments got messed up

Reply:
oh ig u're gonna have to fix it then lol